ui/app/pages/home/home.component.js

```diff
@@ -45,6 +45,8 @@ export default class Home extends PureComponent {
     shouldShowSeedPhraseReminder: PropTypes.bool,
     rejectProviderRequestByOrigin: PropTypes.func,
     isPopup: PropTypes.bool,
+    show3BoxModalAfterImport: PropTypes.bool,
```

Comment:
These props don't appear to be used anymore

Reply:
:+1: